src/coreclr/nativeaot/Runtime/arm64/WriteBarriers.S

```diff
@@ -224,9 +224,11 @@ LEAF_END RhpByRefAssignRefArm64, _TEXT
 
         PREPARE_EXTERNAL_VAR_INDIRECT g_highest_address, x12
         ccmp    x14, x12, #0x2, hs
-        blo     C_FUNC(RhpAssignRefArm64)
+        bhs     LOCAL_LABEL(NotInHeap)
 
-NotInHeap:
+        b       C_FUNC(RhpAssignRefArm64)
```

Comment:
Conditional jump to non-local lables are not allowed in subsections-via-symbols mode. I updated 3 places to use non-conditional branches instead.

Most of the diff is just decorating local labels with `LOCAL_LABEL`.